docker/bin/docker-entrypoint.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+set -e
+
+# Set some vars
+PRE_POP_VOLS=("config" "mods" "art") # these are folders which contain runtime needed files, and need to be represented in the host
+BBS_ROOT=/enigma-bbs # install location
+BBS_STG_P=/enigma-bbs-pre # staging location for pre populated volumes (PRE_POP_VOLS)
```

Comment:
Maybe picky, bit I have a general distaste for opaque variable names. There's no limit in bash, so more descriptive and verbose variable names are preferable for maintainability.

Reply:
can do but just to make sure i understand:
PRE_POP_VOLS=PRE_POPULATED_VOLUMES
BBS_ROOT=BBS_ROOT_DIR
BBS_STG_P=BBS_STAGING_PATH